concept/thing/thing_manager.rs

```diff
@@ -531,8 +486,7 @@ impl ThingManager {
                     .map_err(|err| ConceptReadError::SnapshotIterate { source: err })
             });
 
-        let attribute_value_type_prefix =
-            AttributeVertex::value_type_category_to_prefix_type(ValueTypeCategory::Struct);
+        let attribute_value_type_prefix = Attribute::prefix_for_type(attribute_type.clone());
```

Comment:
The only place it's still needed afaics is in `get_instances_in`. It calls `T::prefix_for_type(thing_type /* : T */)` to create the iterator prefix. Also, because `ObjectType` also impls `ThingType`, it can't just be an associated constant.

Reply:
Right ok